src/Objs/Font/FontComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import { connect, Obj } from 'scrivito'
+import { Helmet } from 'react-helmet-async'
+import { isFont } from './FontObjClass'
+
+// FontComponent is rendered via the DesignAdjustments component
```

Comment:
> Maybe that's what the comment actually wanted to tell us?

The comment was written by me so I wanted to express, that in this file this component is _not_ a `provideComponent` but a regular react component.

I also placed this component in `src/Objs/Font` since it is closely connected to the definition of a font and the editing config.

Reply:
> I also placed this component in src/Objs/Font since it is closely connected to the definition of a font
From my point of view it's more closely connected to the design adjustments. See this comment: https://github.com/Scrivito/scrivito-portal-app/pull/709#discussion_r1880565997.

I think the component here is specific to the design adjustments and not generic for Font objs.

For example, imagine an image gallery/slider component. Would the component for an individual image slide go to src/Objs/Image or to the folder of the slider?